config.json-dist

```diff
@@ -0,0 +1,12 @@
+{
+    "default": {
+        "normandy_url": "https://localhost:8000/api/v1",
+        "api_token": "",
+        "verify": false
+    },
+    "development": {
+        "normandy_url": "https://normandy.dev.mozaws.net/api/v1",
+        "api_token": "",
+        "verify": true
+    }
```

Comment:
I think I would make the default just a name of an environment. Something like this:

```json
{
  "default_env": "local",
  "envs": {
    "local": {
      "normandy_url": "https://localhost:8000/api/v1",
      "api_token": "",
      "verify": false
    },
    "dev": {
      ...
    }
  }
}